src/Infrastructure/LeanCode.ExternalIdentityProviders/ExternalIdentityProvidersModule.cs

```diff
@@ -2,28 +2,77 @@
 using Autofac;
 using LeanCode.Components;
 using LeanCode.ExternalIdentityProviders.Apple;
+using LeanCode.ExternalIdentityProviders.Facebook;
 using LeanCode.ExternalIdentityProviders.Google;
+using LeanCode.Facebook;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace LeanCode.ExternalIdentityProviders
 {
+    [Flags]
+    public enum Providers
+    {
+        Facebook = 0b001,
+        Apple = 0b010,
+        Google = 0b100,
+
+        None = 0b000,
```

Comment:
I'd expect attempting to register this module with `Providers.None` doesn't make sense and should probably throw.